stable/contour/Chart.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: v1
+appVersion: "v0.15.0"
+description: Contour Ingress controller for Kubernetes
+name: contour
+version: 0.1.0
+home: https://projectcontour.io
+keywords:
+  - ingress
+  - envoy
+  - contour
+maintainers:
+  - name: Peter Grant
```

name needs to be your GitHub username

```suggestion
  - name: pickledrick
```